src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions_CreateGetHashCodeMethod.cs

```diff
@@ -179,11 +179,19 @@ public static IMethodSymbol GetBaseGetHashCodeMethod(INamedTypeSymbol containing
                 return thisSymbol;
             }
 
-            var memberType = member.GetSymbolType();
-            var primitiveValue = IsPrimitiveValueType(memberType) && memberType.SpecialType != SpecialType.System_String;
-            var isTupleType = memberType?.IsTupleType == true;
-            if (primitiveValue || isTupleType)
+            if (member.GetSymbolType().IsValueType)
```

Comment:
above code seems to imply that GetSymbolType could return null.  can you be resilient to that?